redbot/cogs/trivia/data/lists/mlb.yaml

```diff
@@ -0,0 +1,258 @@
+AUTHOR: @KaiGucci (kidsterz)
```

Comment:
This is causing an error parsing the YAML. This field cannot start with an `@` symbol.

Reply:
I think it should be good now

Reply:
Nvm there are some more errors